src/lib/zod/index.ts

```diff
@@ -5,14 +5,78 @@ import { zEthAddressOnly } from '../../../src/forms/formHelpers';
 export const createCircleSchemaInput = z
   .object({
     user_name: z.string().min(3).max(255),
-    address: zEthAddressOnly,
     circle_name: z.string().min(3).max(255),
     protocol_id: z.number().int().positive().optional(),
     protocol_name: z.string().min(3).max(255).optional(),
   })
+  .strict()
   .refine(
```

Comment:
@topocount any workaround how i can make the refine work with your zod parser ? fails on CI

Reply:
Sorry, the changes merged to main should work, if you rebase. I updated the type to accept everything except `passthrough` zod parsers.